tonic_validate/metrics/contains_text_metric.py

```diff
@@ -18,7 +18,8 @@ def __init__(
         case_sensitive: Optional[bool] = False,
     ):
         """
-        Create a metric that checks if the LLM response contains a given string.
+        Create a binary metric that checks if the LLM response contains a given string.
```

Comment:
Creates a binary metric that checks whether the LLM response contains a given string.